server/pbenchinacan/pod-nodata.yml

```diff
@@ -155,12 +149,12 @@ spec:
     volumeMounts:
     - mountPath: /var/www/html/dashboard
       name: dashboard_deployment
+    workingDir: /
   volumes:
   - name: dashboard_deployment
     hostPath:
       path: ${PB_DASHBOARD_DIR}
       type: DirectoryOrCreate
-    workingDir: /
```

Comment:
As long as you're making this change to `pod-nodata.yml`, could you please make the same change to `pod.yml`?

Reply:
I've merged them down to one.